client/src/components/ButtonEpisodeCheck.js

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import { ReactComponent as IconCheck } from '../assets/icons/check-circle-solid.svg'
+
+ButtonEpisodeCheck.propTypes = {
+  id: PropTypes.number.isRequired,
+  isEpisodeWatched: PropTypes.bool.isRequired,
+  handleCheckEpisode: PropTypes.func.isRequired,
+}
+
+export default function ButtonEpisodeCheck({
+  id,
+  isEpisodeWatched,
+  handleCheckEpisode,
+}) {
+  return (
+    <StyledButton
+      onClick={() => handleCheckEpisode(id)}
```

Comment:
`handleCheckEpisodes` sounds like a function you would call, however, this function is passed as props, so a better naming convention would be:
```suggestion
      onClick={() => onCheckEpisode(id)}
```
I think we discussed that in the course once, that whenever we pass a function, we like to name it `onSomething` and once we actually call it `handleSomething`. For a better explanation, we should ask Jerry again though 😏

Reply:
Good point, didn't think about that